public/locales/es/common.json

```
@@ -0,0 +1,272 @@
+    "pickleApy": "PICKLE APY",
+    "selectAll": "Elegir todo",
+    "selectBoost": "Elige que granjas potenciar",
+    "selectedFarms": "Granjas seleccionadas",
+    "selectFarms": "Elige a que Granjas distribuir las ganancias en PICKLE usando tu DILL",
+    "selectPrompt": "Por favor elige Granjas desde el desplegable",
+    "submitVote": "Emitir voto",
+    "token": "Token",
+    "totalApyRange": "Rango total de APY",
+    "vote": "Votar",
+    "voting": "Votando...",
+    "weight": "Peso",
+    "weightsInvalid": "Emitir voto (las ponderaciones deben sumar el 100%)"
+  },
+  "info": {
+    "address": "Dirección",
+    "asset": "Activo",
+    "assets": "Activos",
+    "brining": "¿Cómo envasas?",
+    "currentTokenBalance": "Cantidad de tokens depositados en {{ target }}",
+    "currentUsdValue": "Valor actual en USD de {{ target }}",
+    "currentValue": "Valor actual en USD",
+    "daily": "{{ percent }}% diario",
+    "dailyEmission": "Emisiones diarias",
+    "dailyVolume": "Volumen diario",
+    "day": "Día",
+    "dayApy": "APY actual del día",
+    "depositedBalance": "Depositar saldo",
+    "earnedTokens": "Tokens ganados",
+    "earnings": "Ganancias",
+    "emissionRate": "Ratio de emisión: {{ count }} pickle por bloque",
+    "farm": "Granja",
+    "farms": "Granjas",
+    "jar": "Frasco",
+    "jars": "Frascos",
+    "jarValueLocked": "Valor en Frascos",
+    "lastUpdated": "Última actualización",
+    "lifetimeValue": "Valor histórico total en USD",
+    "month": "Mes",
+    "monthly": "{{ percent }}% mensual",
+    "monthlyApy": "APY mensual",
+    "pickleApy": "{{ percent }}% PICKLE APY",
+    "picklePerDay": "Pickle por día",
+    "picklePrice": "Precio de Pickle",
+    "picklePulse": "Pickle Pulse",
+    "polyJars": "polyJars",
+    "tokens": "Tokens",
+    "value": "Valor",
+    "week": "Semana",
+    "weekly": "{{ percent }}% semanal",
+    "weeklyApy": "APY semanal"
+  },
+  "meta": {
+    "description": "Pickle Finance ayuda a los usuarios a maximizar sus ganancias en DeFi auto-invirtiendo sus ganancias, ahorrandoles tiempo y gas.",
+    "title": "El futuro de las finanzas es verde",
+    "titleFull": "Pickle Finance — El futuro de las finanzas es verde"
+  },
+  "nav": {
+    "dill": "Dill",
+    "discord": "Discord",
+    "docs": "Documentación",
+    "feedback": "Feedback",
+    "forum": "Foro",
+    "github": "Github",
+    "info": "Información",
+    "jars": "Frascos",
+    "jarsAndFarms": "Frascos y Granjas",
+    "medium": "Medium",
+    "pickleEth": "PICKLE-ETH",
+    "profit": "Ganancias",
+    "stats": "Estadísticas",
+    "twitter": "Twitter",
+    "vote": "Votar"
+  },
+  "oldFarms": {
+    "back": "a nuevas granjas",
+    "disclaimer": "Estas granjas han parado de ganar $PICKLEs el Jue 22 de Abril de 2021 (12 AM GMT) <br /> Para ganar $PICKLEs, migra cualquier token aquí to a las nuevas granjas <br /> Despliega en cualquier granja donde tengas tokens y haz click en <strong>Migrar</strong>"
```

@mannyreimi88 is this redundant?

```suggestion
    "disclaimer": "Estas granjas han parado de ganar $PICKLEs el Jue 22 de Abril de 2021 (12 AM GMT) <br /> Para ganar $PICKLEs, migra cualquier token aquí a las nuevas granjas <br /> Despliega en cualquier granja donde tengas tokens y haz click en <strong>Migrar</strong>"
```

---

Yes, that to doesn't exists in spanish. Thanks for spotting it